javaslang/src/main/java/javaslang/collection/Arrays2.java

```diff
@@ -71,6 +71,13 @@
         return copy;
     }
 
+    static <T> T[] pushBack(T[] array, T element) {
+        final T[] copy = (T[]) new Object[array.length + 1];
+        System.arraycopy(array, 0, copy, 0, array.length); // why is this a *LOT* slower than prepend???
```

Comment:
Any help would be appreciated here :/

why is 
```java
System.arraycopy(array, 0, copy, 0, array.length);
```
almost twice as fast as
```java
System.arraycopy(array, 0, copy, 0, array.length)
```
?